.storybook/preview.js

```diff
@@ -27,3 +29,23 @@ for (let i = -3; i <= 6; i++) {
 }
 addDecorator(withPaddings);
 addParameters({ paddings });
+
+// Create viewports using widths defined in design tokens
+const breakpointViewports = Object.keys(breakpoints).map((name) => {
+  return {
+    name: `breakpoints.${name}`,
```

Comment:
Missing a dollar sign in the output (should display as `breakpoints.$xs`, see: https://v-next--cloudfour-patterns.netlify.com/?path=/docs/design-tokens-breakpoint--page)

Reply:
Ah, I read over that too quickly. Saw a dollar sign in a literal and went with it. Fixing now.

Reply:
Deployed with this update.